site/docs/v1/tech/core-concepts/_fusionauth_roles.adoc

```diff
@@ -0,0 +1,59 @@
+.FusionAuth application roles
+[cols="1,2,1"]
+|===
+| Name | Id | Description
+
+| `admin` | `631ecd9d8d404c13827780cedb8236e2` | Can manage everything, including creating new users with administrator privileges.
```

Comment:
Should we also show the human readable name we display in the UI? 

Should we also list the human readable role name they will see in the UI?


```
 Global admin (admin)
 API key manager (api_key_manager)
 Application deleter (application_deleter)
 Application manager (application_manager)
 Audit log viewer (audit_log_viewer)
 Connector deleter (connector_deleter)
 Connector manager (connector_manager)
 Consent deleter (consent_deleter)
 Consent manager (consent_manager)
 Email template manager (email_template_manager)
 Event log viewer (event_log_viewer)
 Form deleter (form_deleter)
 Form manager (form_manager)
 Group deleter (group_deleter)
 Group manager (group_manager)
 Key manager (key_manager)
 Lambda manager (lambda_manager)
 Reactor manager (reactor_manager)
 Report viewer (report_viewer)
 System configuration manager (system_manager)
 Tenant deleter (tenant_deleter)
 Tenant manager (tenant_manager)
 Theme manager (theme_manager)
 User action deleter (user_action_deleter)
 User action manager (user_action_manager)
 User deleter (user_deleter)
 User manager (user_manager)
 Webhook manager (webhook_manager)
 ```

Reply:
I didn't think the human readable name added much, since it is pretty much always the same as the regular name (other than for global admin). I have a commit which I can revert which brings them back in if you feel strongly.